webpack.config.js

```diff
@@ -1,13 +1,19 @@
 const path = require('path')
-const SpriteLoaderPlugin = require('svg-sprite-loader/plugin')
 
 module.exports = {
   mode: 'development',
   module: {
     rules: [
       { test: /\.js$/, loaders: ['babel-loader'], exclude: /node_modules/ },
-      { test: /assets\/\w+\.svg$/, loader: 'svg-sprite-loader', exclude: /node_modules/, options: { extract: true } },
-      { test: /SVGIcon\/icons\/\w+\.svg$/, loader: 'svgr/webpack', exclude: /node_modules/, options: { extract: true } },
+      {
+        test: [
```

Comment:
I updated this from `/\.svg/` because I had to specify different loaders, I think we can combine them again